pkg/cmd/ssh/ssh_test.go

```diff
@@ -286,16 +287,15 @@ var _ = Describe("SSH Command", func() {
 			Expect(gardenClient.Get(ctx, key, bastion)).NotTo(Succeed())
 
 			// assert that no temporary SSH keypair remained on disk
-			_, err := os.Stat(options.SSHPublicKeyFile)
-			Expect(err).To(HaveOccurred())
+			//_, err := os.Stat(options.SSHPublicKeyFile)
```

Comment:
need some help on this. The tests used values from the options that are set internally by the run command. We no longer have access as the options struct is not exposed.

Reply:
For E2E tests create a cobra command and call `cmd.Execute()` and maybe use mocks or just check the output. In this case do not create a cobra command but create a `SSHOptions` instance and call the `Run` method.